src/controllers/update/relevant.period.provide.information.now.or.later.controller.ts

```diff
@@ -0,0 +1,47 @@
+import { NextFunction, Request, Response } from "express";
+
+import { logger } from "../../utils/logger";
+import * as config from "../../config";
+import { ApplicationData } from "../../model";
+import { getApplicationData } from "../../utils/application.data";
+import { ProvideInformation } from "../../model/update.type.model";
+
+export const get = async (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `GET ${config.RELEVANT_PERIOD_DO_YOU_WANT_TO_PROVIDE_PRE_REG_INFO_NOW_PAGE}`);
+    const appData: ApplicationData = await getApplicationData(req.session);
+
+    return res.render(config.RELEVANT_PERIOD_DO_YOU_WANT_TO_PROVIDE_PRE_REG_INFO_NOW_PAGE, {
+      backLinkUrl: config.RELEVANT_PERIOD_REQUIRED_INFORMATION_CONFIRM_PAGE + config.RELEVANT_PERIOD_QUERY_PARAM,
+      templateName: config.RELEVANT_PERIOD_DO_YOU_WANT_TO_PROVIDE_PRE_REG_INFO_NOW_PAGE,
+      ...appData,
+    });
+  } catch (error) {
+    logger.errorRequest(req, error);
+    next(error);
+  }
+};
+
+export const post = async(req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `POST ${config.RELEVANT_PERIOD_DO_YOU_WANT_TO_PROVIDE_PRE_REG_INFO_NOW_PAGE}`);
+
+    const provideInformation = req.body[ProvideInformation];
+    const isProvidingInformationNow = provideInformation === "1";
```

Comment:
could we simplify it further by doing
```suggestion
    const isProvidingInformationNow = req.body[ProvideInformation] === "1";
```
Also suggest moving this line down to just above the line of `if (appData.update) {`